### PR TITLE
fix: optimize psql dump test

### PR DESCRIPTION
**Pull request by cristianmtr**
Closes https://github.com/jina-ai/internal-tasks/issues/152
Part of https://github.com/jina-ai/internal-tasks/issues/142

BEFORE MERGING: 

merge commons PR https://github.com/jina-ai/jina-commons/pull/26
and remove https://github.com/jina-ai/executors/pull/99/files#diff-27c5febcc9ce93e8e5341f263b7d68962733b8a2a86c583723d025a1db39d7a3L1

### CODE REVIEW

**Review by numb3r3 — APPROVED** [review]
LGTM

File: jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py [hunk 1]
```diff
@@ -197,3 +212,21 @@ def get_size(self):
...
+        connection = self._get_connection()
+        # always order the dump by id as integer
+        cursor = connection.cursor('generator')  # server-side cursor
+        cursor.itersize = 10000
+        if include_metas:
+            cursor.execute(f'SELECT ID, EMBEDDING, DOC from {self.table} ORDER BY ID')
+            for rec in cursor:
+                yield rec[0], rec[1], rec[2]
+        else:
+            print(f'### NO METAS', flush=True)
```
Comment (maximilianwerk) [inline jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py:228]: remove that print?

File: tests/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,6 @@
 git+https://github.com/jina-ai/jina.git@v2.0.7#egg=jina[hub]
 psycopg2-binary==2.8.6
-git+https://github.com/jina-ai/jina-commons
-pytest
+jina_commons @ git+https://github.com/jina-ai/jina-commons@feat-optional-metas
```
Comment (maximilianwerk) [inline tests/requirements.txt:3]: revert to main

**Review by maximilianwerk — CHANGES_REQUESTED** [review]
to cleanup otherwise LGTM

File: jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py [hunk 3]
```diff
@@ -86,8 +89,10 @@ def use_table(self):
             try:
                 cursor.execute(
                     f'CREATE TABLE {self.table} ( \
-                    ID VARCHAR PRIMARY KEY,  \
-                    DOC BYTEA);'
+                    ID VARCHAR PRIMARY KEY  \
+                    , EMBEDDING BYTEA  \
+                    , DOC BYTEA  \
+                    );'
```
Comment (numb3r3) [inline jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py:95]: This is a breaking change. Could we also release a tag-specific one without changing the table columns?  As, our EAP custom already has millions of docs stored n PostgresStorage.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py (hunk 3 above)
Comment (numb3r3) [reply jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py:95]: I just got some feedbacks from our eap partner, they said they can re-index their data. So I think we are free to release the new psql indexer

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**